packages/foreman/rubygem-sassc/rubygem-sassc.spec

```diff
@@ -70,7 +70,7 @@ cp -a .%{gem_dir}/* \
 mkdir -p %{buildroot}%{gem_extdir_mri}/%{gem_name}
 
 # hack to get the C extension where we expect it to be
-cp %{buildroot}/opt/theforeman/tfm/root/usr/share/gems/gems/sassc-%{version}/ext/libsass.so .%{gem_extdir_mri}/
+cp %{buildroot}/usr/share/gems/gems/sassc-2.2.1/ext/libsass.so .%{gem_extdir_mri}/
```

Comment:
This will break on EL7 since it needs the SCL path present. I think you will need an if-else to handle both.